samples/cpp/benchmark_app/README.md

```diff
@@ -109,18 +109,15 @@ Running the application with the `-h` or `--help` option yields the following us
 ```
 [Step 1/11] Parsing and validating input arguments
 [ INFO ] Parsing input parameters
-
 benchmark_app [OPTION]
-Options:
 
+Options:
```

Comment:
maybe instead of `benchmark_app [OPTION]` we need to highlight something similar to:
```
usage: git log [<options>] [<revision-range>] [[--] <path>...]
   or: git show [<options>] <object>...
```
?

Reply:
I added `usage`. This also aligns how python argparse generates help